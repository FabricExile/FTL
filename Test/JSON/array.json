[]
[43,7.98E+3,-5.12e-27,4.12381823858512,"\"]",{"fred":[5,6]}]
