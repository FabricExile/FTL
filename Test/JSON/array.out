1:1 ARRAY 0
2:1 ARRAY 4
  2:2 INTEGER 43
  2:5 SCALAR 7980
  2:13 STRING 2 '"]'
  2:19 OBJECT 1
    2:20 STRING 4 'fred'
      2:27 ARRAY 2
        2:28 INTEGER 5
        2:30 INTEGER 6
